src/Payments/PaymentMethodsRecord.res

```diff
@@ -172,7 +173,7 @@ let paymentMethodsFields = [
   },
   {
     paymentMethodName: "crypto_currency",
-    fields: [InfoElement],
+    fields: [CurrencyNetworks, InfoElement],
```

Comment:
Why are we hardcoding CurrencyNetworks? Shouldn't this data come from backend

Reply:
Updated